db/schema.rb

```diff
@@ -2,8 +2,8 @@
 # of editing this file, please use the migrations feature of Active Record to
 # incrementally modify your database, and then regenerate this schema definition.
 #
-# This file is the source Rails uses to define your schema when running `rails
-# db:schema:load`. When creating a new database, `rails db:schema:load` tends to
+# This file is the source Rails uses to define your schema when running `bin/rails
```

Comment:
> You should remove any changes to schema.rb from your PR. This file is automatically generated when you run migrations, but it reflects your own database environment and defaults and will be different from what runs on production.

okay is it possible to remove it? because it's already uploaded in my remote branch

Reply:
The easiest way at this point — since there is just a small amount of code changed — is to hard-reset your branch back to `master` and then re-add your other changes. The `docs/git.md` has some quick references for how to do this.

Reply:
> The easiest way at this point — since there is just a small amount of code changed — is to hard-reset your branch back to `master` and then re-add your other changes. The `docs/git.md` has some quick references for how to do this.

alright let me work on it, so this mean i have to delete this PR right?.What if i just create a new branch and add this new changes or it will still have  schema.rb files

Reply:
If you do a hard reset of this branch, you'll basically rewind it to the same state as `master` (and your schema.rb will be the same as from `master`). You can then add a new commit with just the change to the haml file, and then push it to github with the `force` option to overwrite the branch. Then it will show up in this PR.